index.html

```diff
@@ -0,0 +1,1008 @@
+            <a href="#" class="funding-article-title">
+              <img src="images/case-status-funding-article.jpeg" alt="case study funding">
+              <h4 class="fund-title"> Case Status</h4>
+  
+              <div>
+                <h4>$1.5</h4>
+                <h5>SEED FUND</h5>
+              </div>
+            </a>
+  
+            <div class="funding-label">
+              <a href="#">
+                <i class="fa fa-tag label"></i>
+                <span>Real Time, +3</span>
+              </a>
+  
+              <a href="#">
+                <i class="fa fa-briefcase label"></i>
+                <span>BIP Capital</span>
+              </a>
+            </div>
+                <a href="#">Alabama-Based Client Relationship Management Company Cse Status Raises $1.5 Million <span>on Pulse 2.0</span></a>
+              
+            
+          </article>
+          <article class="funding-3 funding">
+            <a href="#" class="funding-article-title">
+              <img src="images/ride-report-funding-article.png" alt="ride report funding">
+              <h4 class="fund-title">Ride Report</h4>
+  
+              <div >
+                <h4>$10M</h4>
+                <h5>SERIES A</h5>
+              </div>
+            </a>
+  
+            <div class="funding-label">
+              <a href="#">
+                <i class="fa fa-tag label"></i>
+                <span>Transportation</span>
+              </a>
+              <a href="#">
+                <i class="fa fa-briefcase label"></i>
+                <span>Homebrew, +3</span>
+              </a>
+  
+            </div>
+  
+            
+            <a href="#">Ride Report Raises $10M in Series A Funding <span>on FinSMEs</span></a> 
+              
+            
+          </article>
+          <article class="funding-4 funding">
+            <a href="#" class="funding-article-title">
+              <img src="images/ride-report-funding-article.png" alt="ride-report funding">
+              <h4 class="fund-title">Harbour BioMed</h4>
+  
+              <div >
+                <h4>$67.4</h4>
+                <h5>SERIES B</h5>
+              </div>
+            </a>
+  
+            <div class="funding-label">
+              <a href="#">
+                <i class="fa fa-tag label"></i>
+                <span>Biotechnology</span>
+              </a>
+              <a href="#">
+                <i class="fa fa-briefcase label"></i>
+                <span>Legend Capital, +8</span>
+              </a>
+  
+            </div>
+  
+            
+            <a href="#">Harbour BioMed raises $67.4M to treat coronavirus & cancer <span>on Silicon Canals - European technology news</span></a> 
+              
+            
+          </article>
+
+        </div>
+
+          
+
+       
+
+        
+
+
+      </section>
+      
+      <section class="categories">
+        <h6 class="section-article-heading">Heading</h6>
+        <article class="category-article">
```

Comment:
- [ ] In the categories section, check the articles' image size when you are in the mobile version. The image should have a large width.

- [ ] The titles below the images should be red and have more padding-left.

- [ ] The gray subtitle should go below the main article's title and should not present where you have it now, please edit it in all articles.

![Screenshot from 2020-03-20 16-21-35](https://user-images.githubusercontent.com/51926937/77210693-3fd23b80-6ac7-11ea-861b-1016655d8361.png)

Original:
![Screenshot from 2020-03-20 16-21-26](https://user-images.githubusercontent.com/51926937/77210703-43fe5900-6ac7-11ea-8a1c-07c593f17bf6.png)